rdflib/namespace.py

```diff
@@ -225,7 +226,7 @@ def __getattr__(self, name):
                 raise AttributeError(e)
 
     def __repr__(self):
-        return f"{self.__module__}.{self.__class__.__name__}({self!r})"
+        return f"{self.__module__}.{self.__class__.__name__}({str(self)!r})"
```

Comment:
`super().__repr__()` works but results in the rather unsightly:

```python
rdflib.namespace.ClosedNamespace(Namespace('http://example.com/name/space/'))
```

And `super(str, self).__repr__()` results in:

```python
rdflib.namespace.ClosedNamespace(<super: <class 'str'>, NULL>)
```